db/seeds.rb

```diff
@@ -5,18 +5,27 @@
 Item.create(name: 'Bone-in Rib Eye', description: "Vegetarianism's downfall", price: 12.99, category_id: 1, image_url: "http://rothmanns2.dtpcreative.com/wp-content/uploads/2014/05/steak-btn.jpg")
-Item.create(name: 'Soda', description: 'Bubbly & definitely not pop', price: 1.59, category_id: 2, image_url: "http://www.wall321.com/thumbnails/detail/20120306/drink%20macro%20ice%20cubes%201920x1200%20wallpaper_www.wall321.com_47.jpg")
+Item.create(name: 'Country Fried Steak', description: "Bring your own fiddle", price: 8.99, category_id: 1, image_url: "http://www.thehautemeal.com/wp-content/uploads/2015/03/Chicken_Schnitz_Main.jpg")
+
 Item.create(name: 'Lemonade', description: 'Made from the lemons life gave you', price: 1.69, category_id: 2, image_url: "http://roodepoortrecord.co.za/wp-content/uploads/sites/20/2015/08/6800363-lemonade-wallpaper-Medium.jpg?2a5d20")
+Item.create(name: 'Bottled Water', description: 'Oh you fancy huh?', price: 1.99, category_id: 2, image_url: "http://news.wtmlondon.com/wp-content/uploads/2015/01/xBottled-Water-800x500_c.jpg.pagespeed.ic.miFiLsM1b6.jpg")
 Item.create(name: 'Beer', description: 'Proof God wants us to be happy', price: 3.99, category_id: 2, image_url: "http://cdn.thedailybeast.com/content/dailybeast/articles/2014/04/05/wine-snobs-there-s-a-beer-for-you/jcr:content/image.crop.800.500.jpg/46659654.cached.jpg")
 Item.create(name: 'Milk', description: 'Grow big and strong', price: 1.99, category_id: 2, image_url: "http://nohealthnolife.net/wp-content/uploads/2015/07/1111.jpg")
-Item.create(name: 'Fries', description: 'Now with no trans fat!', price: 1.99, category_id: 3, image_url: "http://www.foodadvisor.my/attachments/store/fill/800/500/8ab6f5150b3b9b4d70be356908e574932938da53fd55a45a379b593fc649/featured_image.jpg")
-item = Item.create(name: 'Green Beans', description: 'Good and good for you!', price: 2.99, category_id: 3, image_url: "http://postcards.thepurplepassport.com/wp-content/uploads/2013/07/Green-Beans-with-Pork-at-Din-Tai-Fung_7026358825_l-e1372871437478.jpg")
-item2 = Item.create(name: 'Onion Rings', description: 'Deep fried, deep pleasure', price: 2.99, category_id: 3, image_url: "https://lh3.googleusercontent.com/-xT2o5NUey3E/VY1JxkEtufI/AAAAAAAABkE/_hAiOPqRL0A/w800-h800/Spicy_Indian_Style_Onion_Rings_Recipe_thumbnail_1280x800.jpg")
+Item.create(name: 'Soda', description: 'Bubbly & definitely not pop', price: 1.59, category_id: 2, image_url: "http://www.wall321.com/thumbnails/detail/20120306/drink%20macro%20ice%20cubes%201920x1200%20wallpaper_www.wall321.com_47.jpg")
+Item.create(name: 'Old Fashioned', description: 'Put some hair on your chest', price: 4.99, category_id: 2, image_url: "http://vinepair.com/wp-content/uploads/2015/09/old-fashioned.jpg")
```

Comment:
Line is too long. [186/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.